backend/common/models.py

```diff
@@ -0,0 +1,59 @@
+from pydantic import BaseModel
+from typing import List, Optional
+from datetime import datetime
+
+
+class User(BaseModel):
+    id: str
+    username: str
+    password_hash: str
+    email: str
+    full_name: str = ""
+    group_ids: List[str] = []
```

Comment:
in the future we would need a revolut id to facilitate transfers between users

Reply:
Added revolut_id for user